osfoffline/views/rsc/gui/startscreen_gui/startscreen.ui

```diff
@@ -41,45 +41,50 @@
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
-      <x>10</x>
-      <y>20</y>
-      <width>141</width>
+      <x>30</x>
+      <y>50</y>
+      <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
-     <string>Personal Access Token</string>
+     <string>Username (email):</string>
```

Comment:
To be consistent with the OSF homepage, just call this field "Email". (don't leave people guessing about whether a separate username is needed)

<img width="363" alt="screen shot 2015-11-11 at 12 15 45 am" src="https://cloud.githubusercontent.com/assets/2957073/11084451/6fa2d43c-8809-11e5-8012-4510585ee5fc.png">